### PR TITLE
fix: make tutorial work

### PR DESCRIPTION
**Pull request by brycefisher**
### Description 
Tutorial - https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/installing/

Step 3 previously broken due to using deprecated zookeeper connection instead of bootstrap_servers

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docker-compose.yml
Comment (nateab) [inline docker-compose.yml:90]: i dont think we should be hard coding this, was the $KSQL_VERSION substitution not working for you?

**Review by nateab — APPROVED** [review]
Thanks for fixing this! We always appreciate community contributions. Besides my one comment I think this PR looks good, but it might be good to get another set of eyes on it as well

File: docker-compose.yml
Comment (brycefisher) [reply docker-compose.yml:90]: You're correct! I made a different mistake (`docker exec` instead of `docker-compose exec`) that resulted in not having a ptty. I'll update this now!

**Review by nateab — APPROVED** [review]
Sweet thanks for the correction, LGTM!

**Comment by nateab** [comment]
@brycefisher feel free to merge now, since the Jenkins-public-ci passed, the continuous-integration/jenkins/pr-merge is for Confluent employees

**Comment by agavra** [comment]
@nateab - users that aren't part of confluentinc/ksql cannot merge a PR even with our approval I believe. I'll merge this PR

**Comment by nateab** [comment]
@agavra oh no! Got it, thanks for the heads up

**Pull request merged**
